contrib/CLI/CLI11.hpp

```diff
@@ -4843,8 +4843,8 @@ class App {
         set_help_flag("-h,--help", "Print this help message and exit");
     }
 
-    App(const App &) = delete;
-    App &operator=(const App &) = delete;
+    App(const App &) = default;
+    App &operator=(const App &) = default;
 
```

Comment:
What is the reason for this change?

Reply:
Try to fix a clang tidy warning (clang-diagnostic-defaulted-function-deleted).

`dawg/src/dawg_app.h:32:5: warning: explicitly defaulted copy constructor is implicitly deleted [clang-diagnostic-defaulted-function-deleted]`
`    dawg_app(const dawg_app&) = default;        // copy constructor`

`dawg/src/dawg_app.h:34:5: warning: explicitly defaulted move constructor is implicitly deleted [clang-diagnostic-defaulted-function-deleted]`
`    dawg_app(dawg_app&&) = default;             // move constructor`

Reply:
There are good reasons for a library to delete their copy constructors. This warning isn't due toe CLI11. It's due to how you have written `class dawg_app`. Remove those two constructors from dawg_app and revert the changes to CLI11. Let's see if that works.

Reply:
I reverted the changes on CLI11.hpp and fixed it on `class dawg_app`. See [last commit](https://github.com/CartwrightLab/dawg/pull/75/commits/dd9d20ee8b99128e12c82b78e6cba7bc6282f0bc).